tsconfig.json

```diff
@@ -13,6 +13,12 @@
       "test_utils/*": [
         "src/test_utils/public/*"
       ],
+      "plugins/dashboard_embeddable/*": [
```

Comment:
no need for this, all plugins in core_plugins are covered by line 9

Reply:
i'm not sure if line 9 is working as expected

Reply:
yea, it errors if I take this line out.  I think line 9 needs some work.